### PR TITLE
Add information about users needing to be members of the plugdev group.

### PR DESCRIPTION
**Pull request by matoushybl**

### CODE REVIEW

**Comment by Tiwalun** [comment]
I'm not sure if we should add this as a general recoomendation, because in "modern" distributions it should not be necessary. The *uaccess* tag, which is set by the udev rules, should be sufficient.

For example, on my Pop OS installation I'm not a member of the *plugdev* group, but I'm still able to access probes.

Maybe we can say something like:

```
If you're still unable to access the debug probes after following these steps, try adding you user to the plugdev group.
```

**Comment by matoushybl** [comment]
I agree, will reformulate. The thing is though that it happened on Ubuntu server which I'd consider to be somewhat "modern" distribution. And I remember having similar problems on regular Ubuntu as well.

**Comment by matoushybl** [comment]
Updated now. @Tiwalun Thanks for the review!

**Review by Tiwalun — APPROVED** [review]
Thanks!

File: content/docs/getting-started/probe-setup.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ access to the debug probes as well.
 2. Run `udevadm control --reload` to ensure the new rules are used.
 3. Run  `udevadm trigger` to ensure the new rules are applied to already added devices.
 
+If you're still unable to access the debug probes after following these steps, try adding you user to the plugdev group.
```
Comment (Yatekii) [inline content/docs/getting-started/probe-setup.md:32]: Typo :( Well, it's too late now, will PR after.

**Pull request merged**